etc/cn-minimal-test.toml

```diff
@@ -0,0 +1,104 @@
+# service node type, [DN|CN|LOG]
+service-type = "CN"
+
+[log]
+level = "debug"
+format = "json"
+max-size = 512
+
+[cn.frontend]
+#root name
+rootname = "root"
+
+#dump user name
+dumpuser = "dump"
+
+#dump user password
+dumppassword = "111"
+
+#port defines which port the mo-server listens on and clients connect to
+port = 6001
+
+#listening ip
+host = "0.0.0.0"
+
+#host mmu limitation. 1 << 40 = 1099511627776
+hostMmuLimitation = 1099511627776
+
+#guest mmu limitation.  1 << 40 = 1099511627776
+guestMmuLimitation = 1099511627776
+
+#mempool maxsize.  1 << 40 = 1099511627776
+mempoolMaxSize = 1099511627776
+
+#mempool factor.
+mempoolFactor = 8
+
+#process.Limitation.Size.  10 << 32 = 42949672960
+processLimitationSize = 42949672960
+
+#process.Limitation.BatchRows.  10 << 32 = 42949672960
+processLimitationBatchRows = 42949672960
+
+#process.Limitation.PartitionRows.  10 << 32 = 42949672960
+processLimitationPartitionRows = 42949672960
+
+#record the time elapsed of executing sql request
+recordTimeElapsedOfSqlRequest = true
```

Comment:
remove from the toml config. Init with default value in the code

Reply:
I will remove it.

Reply:
I also add a 'cn-full-test.toml' with all parameters with default values.